packages/tooling/fast-tooling-react/src/form/controls/control.linked-data.tsx

```diff
@@ -212,27 +212,71 @@ class LinkedDataControl extends React.Component<
     private handleLinkedDataKeydown = (
         e: React.KeyboardEvent<HTMLInputElement>
     ): void => {
-        if (e.keyCode === keyCodeEnter && e.target === e.currentTarget) {
-            e.preventDefault();
-
-            this.addLinkedData(e.currentTarget.value);
-
-            /**
-             * Adding items to the linked data causes the items to
-             * move the input down while the datalist remains in the same location,
-             * to prevent the datalist from overlapping the input
-             * the datalist is dismissed by defocusing and refocusing the input
-             */
-            (e.target as HTMLElement).blur();
-            (e.target as HTMLElement).focus();
-
-            this.setState({
-                searchTerm: "",
-            });
+        if (e.target === e.currentTarget) {
+            // Enter adds linked data if the input value matches a schema lazily or exactly
+            if (e.keyCode === keyCodeEnter) {
+                e.preventDefault();
+
+                const normalizedValue = e.currentTarget.value.toLowerCase();
+
+                if (
+                    this.lazyMatchValueWithASingleSchema(normalizedValue) ||
+                    this.matchExactValueWithASingleSchema(e.currentTarget.value)
+                ) {
+                    this.addLinkedData(normalizedValue, e.currentTarget.value);
+
+                    /**
+                     * Adding items to the linked data causes the items to
+                     * move the input down while the datalist remains in the same location,
+                     * to prevent the datalist from overlapping the input
+                     * the datalist is dismissed by defocusing and refocusing the input
+                     */
+                    (e.target as HTMLElement).blur();
+                    (e.target as HTMLElement).focus();
```

Comment:
I worry the blur/focus trick could cause strangeness with accessibility.  Won't tools like Narrator announce the focus event and would that be confusing?

Reply:
I've checked other accessibility voice over tools and found no strangeness. I think there's definitely room for improvement, but this single trick doesn't seem to make it worse.